src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -57,6 +59,38 @@
     return ResponseEntity.ok().body(body);
   }
 
+  @ApiOperation(value = "Update a commons")
+  @PreAuthorize("hasRole('ROLE_ADMIN')")
+  @PutMapping("/update")
+  public ResponseEntity<String> updateCommons(
+    @ApiParam("commons identifier") @RequestParam long id,
+    @ApiParam("request body") @RequestBody CreateCommonsParams params
+  ) throws JsonProcessingException
```

Comment:
We can remove `JsonProcessingException` from the method signature if the method cannot throw it.

Reply:
Oooh thank you I forgot I wasn't giving anything in the response body.